fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1120,6 +1168,7 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 				perfContextMetrics->reset();
 			}
 			double commitBeginTime;
+			sharedState->commitQueueLatency->addMeasurement(commitBeginTime - a.startTime);
```

Comment:
commitBeginTime is not initialized yet.

Reply:
Sorry, my bad. As I am resolving conflicts while I cherry-picked the commit from main, I lost it :(.. The main branch has it.  Corrected it now.